src/shims/unix/freebsd/foreign_items.rs

```diff
@@ -47,6 +47,20 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                     this.read_scalar(len)?,
                 )?;
             }
+            "getrandom" => {
+                let [ptr, len, _flags] =
+                    this.check_shim(abi, Abi::C { unwind: false }, link_name, args)?;
+                let ptr = this.read_pointer(ptr)?;
+                let len = this.read_target_usize(len)?;
+                // flags on freebsd does not really matter
```

Comment:
Even when flags are ignored, they must still be `read_` at the appropriate type to ensure some initialized argument of the right size was provided.